doc/releases/changelog-0.23.0.md

```diff
@@ -1,112 +1,170 @@
-  @qml.transforms.transpile(coupling_map=[(0, 1), (1, 2), (2, 3)])
-  def circuit(param):
-      qml.CNOT(wires=[0, 1])
-      qml.CNOT(wires=[0, 2])
-      qml.CNOT(wires=[0, 3])
-      qml.PhaseShift(param, wires=0)
-      return qml.probs(wires=[0, 1, 2, 3])
+  def circuit(x):
+      qml.RX(x, wires=0)
+      qml.RY(0.9, wires=1)
+      qml.RX(0.3, wires=2)
+      qml.CZ(wires=[0, 1])
+      qml.RY(-0.4, wires=0)
+      qml.CZ(wires=[1, 2])
+      return qml.expval(qml.grouping.string_to_pauli_word("ZZZ"))
+  ```
+  ```pycon
+  >>> x = np.array(0.531, requires_grad=True)
+  >>> circuit(x)
+  0.47165198882111165
+  >>> qml.grad(circuit)(x)
+  -0.276982865449393
   ```
 
-* The quantum chemistry functionality is unified in the `qml.qchem` module. The new module provides
-  a differentiable Hartree-Fock solver and contains the functionality to construct a
-  fully-differentiable molecular Hamiltonian. The `qml.qchem` module also provides tools for
-  building other observables such as molecular dipole moment, spin and particle number.
+<h4>QChem unification ⚛️  🏰</h4>
+
+* The quantum chemistry functionality is unified in the `qml.qchem` module
+  providing a differentiable Hartree-Fock solver, the functionality to
+  construct a fully-differentiable molecular Hamiltonian and tools for building
+  other observables such as molecular dipole moment, spin and particle number.
+  [(#2166)](https://github.com/PennyLaneAI/pennylane/pull/2166)
+  [(#2173)](https://github.com/PennyLaneAI/pennylane/pull/2173)
+  [(#2197)](https://github.com/PennyLaneAI/pennylane/pull/2197)
+  [(#2362)](https://github.com/PennyLaneAI/pennylane/pull/2362)
+  [(#2230)](https://github.com/PennyLaneAI/pennylane/pull/2230)
+  [(#2199)](https://github.com/PennyLaneAI/pennylane/pull/2199)
+  [(#2371)](https://github.com/PennyLaneAI/pennylane/pull/2371)
+  [(#2272)](https://github.com/PennyLaneAI/pennylane/pull/2272)
+  [(#2164)](https://github.com/PennyLaneAI/pennylane/pull/2164)
+  [(#2316)](https://github.com/PennyLaneAI/pennylane/pull/2316)
+  [(#2385)](https://github.com/PennyLaneAI/pennylane/pull/2385)
+  [(#2372)](https://github.com/PennyLaneAI/pennylane/pull/2372)
+  [(#2415)](https://github.com/PennyLaneAI/pennylane/pull/2415)
+  [(#2426)](https://github.com/PennyLaneAI/pennylane/pull/2426)
+  [(#2352)](https://github.com/PennyLaneAI/pennylane/pull/2352)
+  [(#2420)](https://github.com/PennyLaneAI/pennylane/pull/2420)
+  [(#2465)](https://github.com/PennyLaneAI/pennylane/pull/2465)
+  [(#2454)](https://github.com/PennyLaneAI/pennylane/pull/2454)
 
   The :mod:`~.qchem` module provides access to a driver function :func:`~.molecular_hamiltonian`
   to generate the electronic Hamiltonian in a single call. For example,
 
   ```python
-  import pennylane as qml
-  from pennylane import numpy as np
-  
   symbols = ["H", "H"]
   geometry = np.array([[0., 0., -0.66140414], [0., 0., 0.66140414]])
   hamiltonian, qubits = qml.qchem.molecular_hamiltonian(symbols, geometry)
```

Comment:
@soranjh I can't quite tell what these two codeblocks are demonstrating. What is different here compared to `qml.qchem` usage in previous PL versions?

Reply:
It might be good to explicitly highlight the differences (in code) here instead.

Reply:
Same goes for the two code blocks below